core/lib/spree/app_configuration.rb

```diff
@@ -220,6 +220,18 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Boolean] Don't escape HTML of product descriptions. (default: +false+)
     preference :show_raw_product_description, :boolean, default: false
 
+    # @!attribute [r] subscriber_names
```

Comment:
I'm not sure this is the right place for these methods. They look like global containers, while here we should only list configurable preferences.

Reply:
I'll move them to `Spree::Config.events`